dto/src/main/java/org/jboss/pnc/dto/response/Graph.java

```diff
@@ -35,10 +37,19 @@
 @JsonDeserialize(builder = Graph.Builder.class)
 public class Graph<T> {
 
+    /**
+     * Map of vertices with the vertex name as a key.
+     */
     private final Map<String, Vertex<T>> vertices;
 
+    /**
+     * List of graph edges.
+     */
     private final List<Edge<T>> edges;
 
+    /**
+     * Map of graph metadata.
+     */
     private final Map<String, String> metadata;
 
```

Comment:
Extend these javadocs to be more descriptive, please.

Reply:
@matejonnet Can you please provide the info I should write there?

Reply:
"Generic map of key-value properties describing the graph."